src/style.css

```diff
@@ -1 +1,83 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
```

Comment:
The `font-family` should include fallback fonts to ensure content is displayed with an alternative font if the main one fails to load.